src/components/va-alert/va-alert.css

```diff
@@ -11,6 +11,33 @@
   background-color: unset;
 }
 
+:host([full-width='true']),
+:host([full-width='']) {
+  border-top-style: solid;
+  border-top-width: 10px;
+}
```

Comment:
If the component has the `full-width` prop, give it a top border. No additional markup required.